client/webserver/site/src/js/charts.js

```diff
@@ -83,8 +83,8 @@ export class DepthChart {
     }
 
     bind(this.canvas, 'wheel', e => { this.wheel(e) })
-    this.resize = h => { this.resize_(h) }
-    bind(window, 'resize', () => { this.resize(parent.clientHeight) })
+    this.resize = () => this.resize_(parent.clientHeight)
+    bind(window, 'resize', this.resize)
```

Comment:
Whoops. This actually broke the markets page a little. We can't hard-code the `resize_` argument, since `resize` is called other places. https://github.com/decred/dcrdex/blob/cfa95ebaa34d954f078fef27289c5232ecb0fbae/client/webserver/site/src/js/markets.js#L302

Reply:
I snuck in a fix into #1051

Reply:
Damn I already put it in 0.2.1.  Where's the diff for that fix?